pkgs/applications/virtualization/xen/packages.nix

```diff
@@ -36,7 +41,10 @@ rec {
     inherit (slim) meta;
```

Comment:
We can add `python3 = unpinnedPython` here to build standalone slim xen with latest python too